src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -179,36 +179,38 @@ const Conversation: React.FC<ConversationProps> = props => {
                   style={{ alignSelf: "center" }}
                   time={messageGroup[0].createdAt}
                   exact
-                  mt={0.5}
                   mb={1}
                 />
                 {messageGroup.map((message, messageIndex) => {
                   const nextMessage = messageGroup[messageIndex + 1]
+                  const senderChanges =
+                    nextMessage && nextMessage.isFromUser !== message.isFromUser
+                  const lastMessageInGroup =
+                    messageIndex === messageGroup.length - 1
+                  const spaceAfter =
+                    senderChanges || lastMessageInGroup ? 2 : 0.5
 
                   return (
-                    <Message
-                      message={message}
-                      initialMessage={conversation.initialMessage}
-                      key={message.internalID}
-                      isFirst={groupIndex + messageIndex === 0}
-                      showTimeSince={
-                        message.createdAt &&
-                        today &&
-                        messageGroup.length - 1 === messageIndex
-                      }
-                      mb={
-                        nextMessage &&
-                        nextMessage.isFromUser !== message.isFromUser
-                          ? 1
-                          : undefined
-                      }
-                    />
+                    <>
+                      <Message
+                        message={message}
+                        initialMessage={conversation.initialMessage}
+                        key={message.internalID}
+                        isFirst={groupIndex + messageIndex === 0}
+                        showTimeSince={
+                          message.createdAt &&
+                          today &&
+                          messageGroup.length - 1 === messageIndex
+                        }
+                      />
+                      <Spacer mb={spaceAfter} />
+                    </>
                   )
                 })}
               </React.Fragment>
             )
           })}
-          <Spacer mb={9} />
+          <Spacer mb={[undefined, 6]} />
```

Comment:
Fixes 7 in 👆 screenshot

Reply:
Interesting you can pass in `undefined` as a value here?

Reply:
yes I recently learned that `null` or `undefined` [skips that breakpoint](https://styled-system.com/guides/array-props)